### PR TITLE
Working with Lists and Dictionaries in Python.

### LINKED ISSUE #1909: Lists and Dictionaries in Python
### Proposed title of article
Working with Lists and Dictionaries in Python

### Introduction paragraph (2-3 paragraphs):
A list is an ordered collection of items. Lists enable you to store a set of items in place. In Python elements of a list are separated by commas and enclosed in square brackets []. 

In Python a dictionary is a collection of key value pairs where each key is connected is a value. In a dictionary order is not preserved compared to lists. A dictionary is wrapped in braces {}, with a series of key value pairs inside the braces. Acknowledging dictionaries empowers you to represent a variety of real-world objects more accurately.

One of the most powerful features of Python readily accessible to new programmers are lists and dictionaries, and they tie together many important concepts in programming. To acquire understanding on lists and dictionaries, this tutorial shows us how to start working with elements in a list, how to access the information once it’s in a dictionary and how to modify that information.

### Key takeaways
By the end of this tutorial, the reader should:
1.	Access, change, add, and remove elements in a list
2.	Sort and Slice a list
3.	Loop through a list 
4.	Access, add and remove new-key value pairs in a dictionary
5.	Loop through a dictionary

### References: 
N/A

### PR DESCRIPTION
**Pull request by duncandegwa**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
This fixes #1909

### CODE REVIEW

File: articles/lists-and-dictionaries-in-python/index.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+}
+```
+
+### Accessing values in a dictionaries
+You can access values using the square brackets notation. To get a value, place its related key inside the square brackets as shown below. 
+
+```python
+stock_1={
+ 'type':'Volvo',
+ 'color':'Black'
```
Comment (geoffrey45) [inline articles/lists-and-dictionaries-in-python/index.md:208]: Use the correct indentation for these code blocks

**Comment by duncandegwa** [comment]
@geoffrey45 I've carried out the proposed change. You can proceed with the review.

**Review by geoffrey45 — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by geoffrey45** [comment]
Hello  @duncandegwa   :wave:

As you know, we were working on a new structure for our EngEd Repo, which led to a slight slow down of article publishing. Everything is almost ready now.

In order to publish your article as soon as possible, please do the following:
- Download the most recent files off this PR (Make sure to pick the latest one with my revisions and edits).
- Delete your article directory .ie. `lists-and-dictionaries-in-python` from the `articles` folders.
- Do a git pull (take a look at the new structure of the repo after a git pull).
- Upload your article files to the [new *articles* folder inside *content*](https://github.com/section-engineering-education/engineering-education/tree/master/content/articles) ie. `/content/articles`.

Ping me via Slack if you have any problems.

**Comment by duncandegwa** [comment]
@geoffrey45  On it ...

**Comment by hectorkambow** [comment]
Thank you v much for your patience as we worked through our Repo re-structuring process. 

The purpose behind this restructure - was to better allow us to handle more volume and allow students to publish more articles and get more involved with the community (with open-source contribution and suggestions). 

With most of the changes coming to completion - we will be sure to get your article published asap. 

Thank you again - and we are looking forward to more amazing articles.

**Comment by hectorkambow** [comment]
great tutorial @duncandegwa very informative and easy to follow along. 

Keep up the great work ! 👍 

cc @geoffrey45

**Pull request merged**